maidroid/script/instruction_set_doc.lua

```diff
@@ -17,7 +17,7 @@ local instr = {
 	{"dig", "<s posname>[, <v errormsg>]", "Digs a node, if posname is a variable, it is set to a bool indicating whether digging succeeded. If false, errormsg tells the reason."},
 	{"place", "<s posname>[, <v errormsg>]", "Places a node, if posname is a variable, it is set to a bool indicating whether placing succeeded. If false, errormsg tells the reason."},
 	{"select_item", "<index>", "Swaps the currently wielded item with the item at place <index> in the main inventory."},
-	{"beep", "", "Execute this every second while the droid walks backwards, pls."},
+	{"beep", "[<n gain>[, <n pitch>]]", "Execute this every second while the droid walks backwards, pls."},
```

Comment:
Ok, getan. Soll man die Lautstärke auch noch in Dezibel statt relativ zum normalen geben?